### PR TITLE
Updates from RHL review of JointCal requirements

### PR DESCRIPTION
**Pull request by TallJimbo**
@RobertLuptonTheGood, here are the changes I've made so far to address your latest comments on the JointCal requirements.

I have also copied some other comments of yours and replied to them on #1; I'd like to discuss a bit there before trying to make the changes here (if still necessary after the conversation).

### CODE REVIEW

**Comment by timj** [comment]
Looking at the PR, it should be pretty straightforward to migrate these changes back to the SysML model.

File: LDM-562.tex [hunk 1]
```diff
@@ -141,7 +143,7 @@ \subsection{Photometric Time-Varying Component}
...
 
 \textbf{Discussion:}
-This component need not be spatially varying on scales below an individual exposure, and it will nearly always be provided by a synthesis of auxilliary data (not fit by JointCal).  In that case, this will be take the form of a number of given data points that should be interpolated to yield the variation with time (sample points may not correspond to the observations of individual exposures).  When fit directly by JointCal, a linear or quadratic function of time should be fit.
+This component need not be spatially varying on scales below an individual exposure (that variation is covered by \ref{DMS-L2JC-REQ-0008}), and it will nearly always be provided by a synthesis of auxilliary data (not fit by JointCal).  In that case, this will be take the form of a number of given data points that should be interpolated to yield the variation with time (sample points may not correspond to the observations of individual exposures).  When fit directly by JointCal, a linear or quadratic function of time should be fit.
```
Comment (RobertLuptonTheGood) [inline LDM-562.tex]: We need to specify that the interpolation scheme provided by the "auxilliary [sic] data" be provided along with the data.

**Review by RobertLuptonTheGood — APPROVED** [review]
Looks good.  One minor comment

File: LDM-562.tex (hunk 1 above)
Comment (timj) [reply LDM-562.tex]: @TallJimbo how do you want to respond to @RobertLuptonTheGood?

Comment (TallJimbo) [reply LDM-562.tex]: Just updated the text.  Sorry I didn't do so previously.

**Pull request merged**